streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/TaskAssignor.java

```diff
@@ -40,10 +40,14 @@
         }
 
         TaskAssignor<C, T> assignor = new TaskAssignor<>(states, tasks, seed);
+        log.info("Assign: sameTopology: {}, states: {}, tasks: {}, replicas: {}",
```

Comment:
I feel I want a INFO level logging statement on whether the clients are the same as before, the rebalancing is not a frequently occurred event, have those logging will not pollute the log file.  People rarely turn on DEBUG level logging.